recipes/otsvm/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1i" %}
+
+package:
+  name: otsvm
+  version: {{ version }}
+
+source:
+  fn: otsvm-{{ version }}.tar.gz
+  url: https://github.com/openturns/otsvm/archive/v{{ version }}.tar.gz
+  sha1: b03b35e0846b7e8b9d75d8fab96ba06baae3d8f1
+
+build:
+  number: 0
+  skip: true  # [py3k or win]
```

Comment:
That's pretty weird as it definitely says it should be [skipped]( https://github.com/conda-forge/openturns-feedstock/blob/ed63aa2ae816fc8916f4885d3ee2834aa496fc01/recipe/meta.yaml#L15 ).